doc/reference/common-weaknesses/state-explosion.rst

```diff
@@ -0,0 +1,15 @@
+State-space explosion due to unconfirmed transactions
+=====================================================
+
+When a contract submits a set of independent transactions, any subset might eventually be confirmed, potentially resulting in an explosion of possible blockchain states.
+
+Risks
+~~~~~
+
+Contracts that submit an unbounded number of transactions, without waiting to be sure they are confirmed, may simply be infeasible to test thoroughly—or to understand—because the state space is so large.
```

Comment:
This doesn't actually point to a risk of anything going bad with the application. "Because the application is much harder to test, this increases the risk of realy weaknesses going undetected."